nilearn/datasets/func.py

```diff
@@ -785,9 +786,8 @@ def fetch_localizer_contrasts(
                 name_aux = str.replace(
                     str.join("_", [data_type, contrast]), " ", "_"
                 )
-                file_path = Path(
-                    "brainomics_data", subject_id, f"{name_aux}.nii.gz"
-                )
+                file_path = f"brainomics_data/{subject_id}/{name_aux}.nii.gz"
```

Comment:
I'm sure you had a good reason but, for my own understanding, for revert to hardcoded `"/"`?

Reply:
hmm... I thing I was getting some tests failures and tried to fix it but forgot to revert those.
let me fix this